packages/cli/src/bin/dev-portal/PersonSideSheet/sheets/FeatureTogglerPortal.tsx

```diff
@@ -0,0 +1,37 @@
+import { useFrameworkFeatureFlags } from '@equinor/fusion-framework-react-app/feature-flag';
```

Comment:
this should be imported from `@equinor/fusion-framework-react/feature-flag`

Reply:
Ok.